api/models/heuristics/directpresentation.py

```diff
@@ -0,0 +1,51 @@
+from models.heuristic import Heuristic
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.chrome.options import Options
+from boilerpipe.extract import Extractor
+from multiprocessing import RLock
+from bs4 import BeautifulSoup, SoupStrainer
+import re
+
+# Procedural 2q
+# Ensure link to EULA can be found on home page
+class DirectPresentation(Heuristic):
+    def score(self, eula):
+        if eula.url is None:
+            return {'score': -1, 'max': 4, 'reason': 'no url'}
+        else:
+            url = eula.url
+            suffixes = [".com", ".net", ".org"]
+            for i in suffixes:
+                n = url.find(i)
+                if n != -1:
+                    break
+            url = url[0:n+4]
+
+            chrome_options = Options()
+            chrome_options.add_argument("--hide-scrollbars")
+            chrome_options.set_headless()
+
+            # Start chrome driver, and set window to initial width and height
+            driver = webdriver.Chrome(chrome_options=chrome_options)
```

Comment:
We should think about using [Scrapy](https://scrapy.org/) instead of chrome unless we need to render the page.  Chrome headless has a huge memory overhead and high cpu load.